packages/sdk/src/sdk/services/Solana/programs/types.ts

```diff
@@ -98,9 +100,15 @@ export const BuildTransactionSchema = z
         z.object({
           /**
            * Specify the precise percentile (0-100) of recent priority fees
-           * to use as this transactions priority fee per compute unit.
+           * to use as this transaction's base priority fee per compute unit.
            */
           percentile: z.number().min(0).max(100),
+          /**
+           * Multiply the base priority fee per compute unit by some factor.
+           * For example a multiplier of 2 and percentile of 50 will set the
+           * priority fee to twice the median.
+           */
+          multiplier: z.number().min(0).optional(),
```

Comment:
defaults to 1

Reply:
Will probably set it back to 1 in a couple weeks